### PR TITLE
[IMP] event: remove 96x134mm badge format

### PR DESCRIPTION
**Pull request by loouis-t**
in this [PR](https://github.com/odoo/odoo/pull/179903) configured two badge formats for the EPSON C4000e printer:
- 96x82mm,
- 96x134mm.

As we don't have a dedicated proper template for the 96x134mm format, we remove it.

In addition, we added the badge type (e.g. VIP, Standard, ...) in the notification displayed once a badge is printed.

Tasks: 4231455, 4229143

### CODE REVIEW

File: addons/event/models/event_registration.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import os
 
 from odoo import _, api, fields, models, SUPERUSER_ID
-from odoo.addons.event.tools.esc_label_tools import print_event_attendees, setup_printer, layout_96x82, layout_96x134
```
Comment (MaxTheWhale) [inline addons/event/models/event_registration.py:7]: You can also remove this from `esc_label_tools`

File: addons/event/models/event_event.py [hunk 2]
```diff
@@ -241,7 +241,6 @@ def _default_question_ids(self):
             ('A6', 'A6'),
             ('four_per_sheet', '4 per sheet'),
             ('96x82', '96x82mm (Badge Printer)'),
-            ('96x134', '96x134mm (Badge Printer)')
```
Comment (MaxTheWhale) [inline addons/event/models/event_event.py:244]: What happens when we remove a selection option in stable (can we?)
Does it need an upgrade script?

File: addons/event/static/src/client_action/event_registration_summary_dialog.js [hunk 3]
```diff
@@ -182,7 +182,11 @@ export class EventRegistrationSummaryDialog extends Component {
         const [{ id: reportId }] = await this.orm.searchRead("ir.actions.report", [["report_name", "=", reportName]], ["id"]);
 
         this.notification.add(
-            _t("'%(name)s' badge sent to printer '%(printer)s'", { name: this.registration.name, printer: this.selectedPrinter.name }),
+            _t("'%(name)s' %(type)s badge sent to printer '%(printer)s'", {
+                name: this.registration.name,
+                type: this.registration.ticket_name,
```
Comment (MaxTheWhale) [inline addons/event/static/src/client_action/event_registration_summary_dialog.js:187]: Ticket type isn't required, so `ticket_name` could be `false`, we should check it first before including it in the message

File: addons/event/models/event_event.py (hunk 2 above)
Comment (qle-odoo) [reply addons/event/models/event_event.py:244]: you cannot change model in stable

Comment (loouis-t) [reply addons/event/models/event_event.py:244]: okay, will retarget to master then

**Comment by loouis-t** [comment]
Retargeted to `master`
See: [https://github.com/odoo/odoo/pull/185446](https://github.com/odoo/odoo/pull/185446)

**Pull request closed without merging**